src/Components/Nav/Nav.js

```diff
@@ -1,8 +1,102 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ITEMTYPE from './ItemType';
 import './Nav.scss';
 
 function Nav() {
-  return <div>Hello</div>;
-}
+  // 통신
+  const [items, setItems] = useState({});
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/NavItemType.json')
```

Comment:
`http://localhost:3000`은 생략 가능합니다. 포트 번호(3000, 3001..)는 로컬 서버를 열 때마다 변경되기 때문에 이를 생략하지 않을 경우 통신 에러가 발생할 수 있습니다. 생략해주세요!
```suggestion
    fetch('/data/NavItemType.json')
```